docs/source/UserGuide/VUDRM-Admin.md

```diff
@@ -0,0 +1,136 @@
+# USER GUIDE
+
+Our VUDRM Admin page gives you the tools you will need to test our DRM services with your content. If you need assistance or clarification in relation to anything in this guide, please contact <support@vualto.com>.
+
+## Login
+
+To access the VUDRM Admin site, please go to the [Admin Login page](https://admin.vudrm.tech/Account/Login) - login using the credentials on your onboarding documentation.
+
+<img src="../_static/user-guide-images/1login.png" class="image" width="500" alt="VUALTO Admin - Login Page"/>
+
+## Dashboard
+
+Once you have successfully logged in, you will be redirected to the `Dashboard` page.
+
+<img src="../_static/user-guide-images/2dashboardpage.png" class="image" width="750" alt="VUALTO Admin - Dashboard Page"/>
+
+The dashboard will show you an overview of all DRM licenses served with your content.
+
+By clicking on the `Range` dropdown field, you can set custom time frames to get an overview for that time period.
+
+<img src="../_static/user-guide-images/3rangedropdown.png" class="image" width="750" alt="VUALTO Admin - Range Dropdown"/>
+
+## Configuration
+
+### Client Info
+
+The `Client Info` section contains the details of how your account is configured, such as which DRM providers are enabled and which URLs have been set to retrieve each service's DRM licenses.
+
+<img src="../_static/user-guide-images/4clientinfo.png" class="image" width="750" alt="VUALTO Admin - Client Info Page"/>
```

Comment:
Can you blur our the redacted fields instead of using a white box as that's the way it is done in other parts of the docs.

Reply:
Actioned.